### DISCUSSION TITLE
Updates regarding the initiative

### DISCUSSION THREAD

**Discussion by KL13NT** [KL13NT/AIUX#14] [General]
I started this initiative back in August 2020 with an extremely vague idea and a lot of passion towards its goals. This year I plan to get back on track with this initiative and have updated its details and goals to be more specific, accurate, and helpful. I also updated all documentation to include [English versions](https://github.com/KL13NT/AIUX/issues/8) as the main language for all documents, issues, and discussions. This is to allow users outside the exclusive Arabic-speaking community to help. I've also added the idea of factions. Each faction represents a collection of goals and responsibilities. Please take a look around and provide feedback on opportunities for improvements.

<div dir="rtl">

لقد بدأت هذه المبادرة في أغسطس 2020 بدون سابق تخطيط ولكن بشغف كبير تجاه أهدافها. أخطط هذا العام للعودة إلى المسار الصحيح مع هذه المبادرة وقد بدأت بتحديث تفاصيلها وأهدافها لتكون أكثر تحديدًا ودقة وإفادة. لقد قمت أيضًا بتحديث جميع الوثائق لتضمن [النسخ الإنجليزية](https://github.com/KL13NT/AIUX/issues/8) كلغة رئيسية لجميع المستندات والقضايا والمناقشات. هذا للسماح لمستخدمين اكثر (خارج المجتمع الناطق باللغة العربية) بالمساعدة. لقد أضفت أيضًا فكرة الفصائل. يمثل كل فصيل مجموعة من الأهداف والمسؤوليات. يرجى إلقاء نظرة على الوثائق الجديدة وتقديم ملاحظاتكم حول فرص التحسين.

</div>

**Comment by TawfikYasser** [comment]
@KL13NT 
<div dir="rtl">
في البداية انا شوفت منصات كتير على سبيل المثال المنصات الهندية بتوفر محتوى تعليمي لايقل ابداً عن محتوى الجامعات، مجاناً ومتاح في اي وقت وعلى اعلى مستوى من التحضير والاهتمام بالطلاب وما الا ذلك زي منصة Unacademy

ف انا بقول ليه ميبقاش في منصة زي دي عربية بتجمع افضل محتوى عربي، مش بس مجرد تجميع فيديوهات يوتيوب وعرضها في مكان واحد، لا الافضل ان الناس اللي بتدرس المحتوى ده على يوتيوب يكون في تعاقد معاهم انهم كلهم يكونوا في منصة واحدة او مكان واحد ويكون في تنظيم وتخطيط لعرض محتواهم القيم ده واكيد هيكون في مصدر للربح علشان الناس دي مش هتشتغل ببلاش وبردو هيكون مش غالى علشان العلم ده بنحاول نوصله لكل الناس، طبعاً الموضوع محتاج تخطيط وتفكير كتير ويبدء صح علشان فعلا لو في حاجة زي كده في المجتمع العربي بتتكلم عن كل حاجة مش بس لغات برمجة او ادوات ولكن كل حاجة محتاجها اي اداة عايز تتعلمها تلاقي ليها حد بيشرحها بلغتك العربية بجودة محترمة ، اظن ده هيعوض قلة الجودة في الجامعات اللي كلنا عارفينها وهتكون فعلا طفرة كبيرة علشان انا شايف ان اهم اسباب ضعف الطلاب الخريجين زي ما بيتقال على السوشيال ميديا جزء كبير منه ضعف مستوى اعضاء هيئة التدريس والطريقة الغلط اللي بيستعملوها والمشاكل الكتير اللي كلنا عارفينها ك طلاب ، لان انا لو طالب لاقيت حد بيشرحلي هياكل بيانات و خوارزميات بطريقة كويسة جدا و عربي ومتاح ف اي وقت مش هيكون في حد يقولي لا مش هتاخد الslides وكمان دلوقتي احنا معندناش مشكلة في التعليم الذاتي ومبقاش مشكلة زي الاول 
اتمنى اننا نبدء فعلا علشان انا متفائل ان ك عرب احنا فعلا بدأن نتحسن ، مش عايزين كل الناس الكويسة تسافر برا وتسيب البلد وعلمهم يكون للغرب 
الموضوع لسه فيه كلام كتير واسف على الاطالة
</div>

**Reply by ghost** [reply]
الأستاذ إبراهيم عادل ( برنامج 
zamerican English + Taleek 
بدأ في الموضوع ده وخلال الشهور الجاية ممكن تطلع الحاجة للنور
هيكون في تعاون مع أسامة الزيرو و الناس اللي شغالة بنفس طريقته 
+ إسلام محفوظ بدأ شعل في
 freecodecamp
 و هيكونوا شغالين على عمل محتوى عربي 
 إن شاء الله
